app.js

```diff
@@ -140,7 +140,9 @@ if (redisUrl) {
 const SIX_HOURS = 1000 * 60 * 60 * 6;
 app.use(session({
 	cookie: {
-		// TODO ...cookieDefaults,
+		httpOnly: true,
+		sameSite: Configuration.get('SESSION_COOKIE_SAME_SITE'),
+		secure: 'auto',
```

Comment:
I think this should be a boolean

Reply:
@virgilchiriac true documentation states you can set it to 'auto'
https://github.com/expressjs/session?tab=readme-ov-file#cookiesecure

> The cookie.secure option can also be set to the special value 'auto' to have this setting automatically match the determined security of the connection